geps/gep-713/index.md

```diff
@@ -3,1155 +3,1192 @@
+| -------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------- | -------------------------------------------- | ---------------- | ------------- |
+| **EnvoyFilter**            | Customize the Envoy configuration generated by istiod, e.g. modify values for certain fields, add specific filters, or even add entirely new listeners, clusters. | GatewayClass, Gateway, Service, ServiceEntry | Custom           | Inherited     |
+| **RequestAuthentication**  | Define request authentication methods supported by a workload.                                                                                                    | GatewayClass, Gateway, Service, ServiceEntry | Custom           | Inherited     |
+| **AuthorizationPolicy**    | Enable access control on workloads in the mesh.                                                                                                                   | GatewayClass, Gateway, Service, ServiceEntry | Custom           | Inherited     |
+| **WasmPlugin**             | Extend the functionality provided by the Istio proxy through WebAssembly filters.                                                                                 | GatewayClass, Gateway, Service, ServiceEntry | Custom           | Inherited     |
+| **Telemetry**              | Defines how telemetry (metrics, logs and traces) is generated for workloads within a mesh.                                                                        | GatewayClass, Gateway, Service, ServiceEntry | Custom           | Inherited     |
+
+#### NGINX Gateway Fabric
+
+<small>https://docs.nginx.com/nginx-gateway-fabric/overview/custom-policies/</small>
+
+Gateway API implementation that supports Gateway API’s `BackendTLSPolicy` as well as the following kinds of policies:
+
+| Policy kind                | Description                                              | Target kinds                  | Merge strategies   | Policy class |
+| -------------------------- | -------------------------------------------------------- | ----------------------------- | ------------------ | ------------ |
+| **ClientSettingsPolicy**   | Define settings related to tracing, metrics, or logging. | Gateway, HTTPRoute, GRPCRoute | Patch defaults     | Inherited    |
+| **ObservabilityPolicy**    | Configure connection behavior between client and NGINX.  | HTTPRoute, GRPCRoute          | None               | Direct       |
+| **UpstreamSettingsPolicy** | Configure connection behavior between NGINX and backend. | Service                       | None               | Direct       |
+
+#### Gloo Gateway
```

Comment:
Please also include https://github.com/kgateway-dev/kgateway, which is the next gen version of Gloo

Reply:
Do you have a link that lists all the policies you support?